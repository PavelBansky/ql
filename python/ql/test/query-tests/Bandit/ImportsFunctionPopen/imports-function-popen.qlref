Bandit/imports-function-popen.ql