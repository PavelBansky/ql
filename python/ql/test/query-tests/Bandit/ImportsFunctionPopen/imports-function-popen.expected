| Test.py:6:14:6:32 | __import__() | Consider possible security implications associated with Popen module |
| Test.py:19:5:19:33 | Attribute() | Consider possible security implications associated with Popen module |
